packages/fields/src/types/Content/Implementation.js

```diff
@@ -13,6 +15,92 @@ function flattenBlockViews(block) {
   ];
 }
 
+/**
+ * @param data Object For example:
+ * {
+ *   document: [
+ *     { object: 'block', type: 'cloudinaryImage', data: { _mutationPath: 'cloudinaryImages.create[0]' },
+ *     { object: 'block', type: 'cloudinaryImage', data: { _mutationPath: 'cloudinaryImages.create[1]' },
+ *     { object: 'block', type: 'relationshipUser', data: { _mutationPath: 'relationshipUsers.create[0]' } }
+ *     { object: 'block', type: 'relationshipUser', data: { _mutationPath: 'relationshipUsers.connect[0]' } }
+ *   ],
+ *   cloudinaryImages: {
+ *     create: [
+ *       { data: { image: <FileObject>, align: 'center' } },
+ *       { data: { image: <FileObject>, align: 'center' } }
+ *     ]
+ *   },
+ *   relationshipUsers: {
+ *     create: [{ data: { id: 'abc123' } }],
+ *     connect: [{ id: 'xyz789' }],
+ *   },
+ * }
+ */
+async function processSerialised({ document, ...nestedMutations }, blocks, graphQlArgs) {
```

Comment:
We're on the server, receiving some already-serialized data that we need to process for potential mutations (connects/creates).